tests/unit/references/test_processors.py

```diff
@@ -120,3 +123,31 @@ def test_reference_builder_add_uid():
     }
 
     assert expected == rb.obj
+
+
+def test_refextract_authors():
+    author_strings = [
+        'Butler, D., Demarque, P., & Smith, H. A.',
+        'Cenko, S. B., Kasliwal, M. M., Perley, D. A., et al.',
+        'J. Kätlne et al.', # Also test some unicode cases.
+        u'J. Kätlne et al.',
+        'Hoaglin D. C., Mostellar F., Tukey J. W.',
+        'V.M. Zhuravlev, S.V. Chervon, and V.K. Shchigolev',
+        'Gómez R, Reilly P, Winicour J and Isaacson R'
+    ]
+
+    expected = [
+        ['Butler, D.', 'Demarque, P.', 'Smith, H. A.'],
+        ['Cenko, S. B.', 'Kasliwal, M. M.', 'Perley, D. A.'],
+        ['J. Kätlne'],
+        ['J. Kätlne'],
+        ['Hoaglin D. C.', 'Mostellar F.', 'Tukey J. W.'],
+        ['V.M. Zhuravlev', 'S.V. Chervon', 'V.K. Shchigolev'],
```

Comment:
Mmh, does it mean you actually don't try to put twist these name into "surname, firstnames"?
Would be great if you can do this as well, since this is the way they are then encoded in the system.